benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/types/bytes/BytesUnit.scala

```diff
@@ -0,0 +1,13 @@
+package cloud.benchflow.dsl.definition.types.bytes
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-23
+ */
+object BytesUnit extends Enumeration {
```

Comment:
This is to be compliant with Docker Compose `v2` standard. We have the following issue to track how to improve it #377